### PR TITLE
feat: unlockable drop landing page

### LINKED ISSUE #6205: unlockable drop landing page
Update of how to display live campaign on the landing page - thinking about adding a new link to the navbar when there will be a drop page live

landing page - main

updated in figma file: https://www.figma.com/file/JXNtryaNYhY0XJNArzvPQx/Handoff?type=design&node-id=394%3A6126&t=GnTXW19oyHFxU0Gz-1

_Originally posted by @exezbcz in https://github.com/kodadot/nft-gallery/issues/6117#issuecomment-1572681703_
            

- #6163

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Needs Design check

- @exezbcz please review

## Context

- [x] Closes #6205
- [ ] Requires deployment <snek/rubick/worker>

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

## Screenshot 📸

- [x] My fix has changed UI

image

### <samp>🤖 Generated by Copilot at c01bff2</samp>

This pull request introduces a new feature for minting NFTs with unlockable content. It adds a new component `UnlockableLandingTag` that shows a tag for unlockable NFTs on the landing page and allows users to filter by this criterion. It also defines a constant `enableUnlockableMintPrefix` that specifies which prefixes can use this feature.

### <samp>🤖 Generated by Copilot at c01bff2</samp>

> _Oh, we're the minters of the `stmn` prefix_
> _We've got a feature that will make you perplex_
> _We can hide some content in our NFTs_
> _And reveal it later with a special key_

### CODE REVIEW

**Comment by exezbcz** [comment]
would be nice to have on every network landing page no? Would it be possible?

**Comment by Jarsen136** [comment]
> would be nice to have on every network landing page no? Would it be possible?

It's possible, but I'm concerned that the user may feel strange seeing the unlockable minting button on other chains except stmn chain. Will we support unlockable minting on other chinas in the future?

**Comment by exezbcz** [comment]
@Jarsen136 Yes. The thing is, they will only know about the drop if they are connected to stmn. Thus I will put it everywhere - the only issue I see on this is that we need a multichain profile, so if the user wants to find it, it takes some time.

**Comment by Jarsen136** [comment]
> would be nice to have on every network landing page no? Would it be possible?

✅ DONE

File: components/collection/unlockable/UnlockableLandingTag.vue [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+              result="shape" />
+            <feGaussianBlur
+              stdDeviation="7.5"
+              result="effect1_foregroundBlur_394_6126" />
+          </filter>
+        </defs>
+      </svg>
+
+      <span> Free Unlockable Mint Live </span>
+      <svg
```
Comment (roiLeo) [inline components/collection/unlockable/UnlockableLandingTag.vue:40]: `NeoIcon`?

File: components/collection/unlockable/UnlockableLandingTag.vue [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        viewBox="0 0 15 2"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg">
+        <line y1="1" x2="15" y2="1" stroke="#CCCCCC" />
+      </svg>
+    </div>
+
+    <nuxt-link
+      class="is-flex is-align-items-center has-text-weight-bold"
+      :to="`/stmn/unlockable`">
```
Comment (roiLeo) [inline components/collection/unlockable/UnlockableLandingTag.vue:53]:
```suggestion
      to="/stmn/unlockable">
```

File: components/collection/unlockable/UnlockableLandingTag.vue [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      Take Me There!
```
Comment (roiLeo) [inline components/collection/unlockable/UnlockableLandingTag.vue:54]: Translation?

File: components/collection/unlockable/UnlockableLandingTag.vue [hunk 4]
```diff
@@ -0,0 +1,71 @@
+<template>
+  <div
+    v-if="isUnlockableLandingTagVisible"
+    class="unlockable-landing-tag is-flex border is-justify-content-space-between px-4 mt-6 mx-4">
+    <div class="is-flex is-align-items-center">
+      <svg
+        class="mr-2"
```
Comment (roiLeo) [inline components/collection/unlockable/UnlockableLandingTag.vue:7]:
```suggestion
```
I don't think it's needed compared to figma file

File: components/collection/unlockable/UnlockableLandingTag.vue [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+              in2="BackgroundImageFix"
+              result="shape" />
+            <feGaussianBlur
+              stdDeviation="7.5"
+              result="effect1_foregroundBlur_394_6126" />
+          </filter>
+        </defs>
+      </svg>
+
+      <span> Free Unlockable Mint Live </span>
```
Comment (roiLeo) [inline components/collection/unlockable/UnlockableLandingTag.vue:39]: Translation?

**Review by roiLeo — APPROVED** [review]
Small stuff otherwise lgtm ✅

File: locales/en.json [hunk 6]
```diff
@@ -477,6 +477,8 @@
     "unlockable": {
       "phaseIntroduction": "We will have a 60-minute window every phase, each featuring only 10 exclusive items.",
       "readyIn": "Your mint will be ready in {0}, DO NOT CLOSE THIS WINDOW",
+      "mintLive": "Free Unlockable Mint Live",
+      "takeMe": " Take Me There!",
```
Comment (roiLeo) [inline locales/en.json:481]: lol is the space forced here? if so it would be better to use `&nbsp;`

**Review by roiLeo — APPROVED** [review]

File: components/collection/unlockable/UnlockableLandingTag.vue (hunk 1 above)
Comment (roiLeo) [reply components/collection/unlockable/UnlockableLandingTag.vue:39]: https://fontawesome.com/search?q=horizontal-rule&o=r ?

Comment (Jarsen136) [reply components/collection/unlockable/UnlockableLandingTag.vue]: They are not exactly the same but ok. Thanks!

File: locales/en.json (hunk 6 above)
Comment (Jarsen136) [reply locales/en.json]: It's a mistake : )

File: components/collection/unlockable/UnlockableLandingTag.vue (hunk 1 above)
Comment (exezbcz) [reply components/collection/unlockable/UnlockableLandingTag.vue]: I think you can use a text "–"

does not have to be completely the same, thanks!

**Comment by exezbcz** [comment]
>thinking about adding a new link to the navbar when there will be a drop page live


image

**Comment by yangwao** [comment]
image

🥹

**Comment by Jarsen136** [comment]
> > thinking about adding a new link to the navbar when there will be a drop page live
> 
Oh, I missed this one. Have added it.
image

File: components/Navbar.vue [hunk 7]
```diff
@@ -52,6 +52,22 @@
...
+              fill="#FF7AC3" />
+          </svg>
```
[the remark this answers is not in the document]
Comment (Jarsen136) [reply components/Navbar.vue:68]: @exezbcz wdyt to replace icon with this one. It looks a little different for me.

https://fontawesome.com/icons/fire-alt?f=classic&s=solid&pc=%23dd83b7

image

[the remark this answers is not in the document]
Comment (exezbcz) [reply components/Navbar.vue:68]: you can use sharp one, does not matter if its not 1:1

image

[the remark this answers is not in the document]
Comment (Jarsen136) [reply components/Navbar.vue]: replaced 
image

**Comment by yangwao** [comment]
legit
image

pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 4.32 USD/DOT ~ 11.574 $DOT
🧗 [16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z](https://polkadot.subscan.io/account/16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)
🔗 [0x88992340031eabd7aec6f07b23ef432ad9f7efb89a88470e3dfd9322727ee3f5](https://polkadot.subscan.io/extrinsic/0x88992340031eabd7aec6f07b23ef432ad9f7efb89a88470e3dfd9322727ee3f5)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**